tools/release/image-mirror/mirror.go

```diff
@@ -210,9 +163,9 @@ func getSleepTime(rateLimitReset string, now time.Time) time.Duration {
 	return calculatedSleepTime
 }
 
-func tagInAllowlist(tag string, allowlist []string) bool {
+func tagInAllowlist(tag RepositoryTag, allowlist []string) bool {
```

Comment:
This change doesn't have to be made. Just pass in `tag.Name` instead.

Reply:
The point being don't change the API of a function if you don't have to. I don't think this is a good practice.

Reply:
I Agree